### ISSUE TITLE
Logs don't get cleared

### ISSUE THREAD

**Issue by lexika979** [doonv/bevy_dev_console#32]
## bevy_dev_console version

533603f4838b7379a9ef773d5bc7f50efb3089af

## What you did

Simply log alot of messages

## What went wrong

The messages dont get cleared no matter the amout logged, causing the fps to rapidly decline over time

**Comment by doonv** [comment]
Yeah this is a problem that I should fix, but whats more concerning is that you said it reduces your fps, which definitely shouldn't happen.

Does the fps decline only happen when the console is open?

**Comment by lexika979** [comment]
Yes, it only drops fps when the console is open

**Comment by doonv** [comment]
Yeah I assume that's because the console renders **all** logs, no matter if you can see them or not. I'll probably try to implement some kind of system to only render the logs you can see
